openpos-peripheral/src/main/java/org/jumpmind/pos/print/EscpPOSPrinter.java

```diff
@@ -465,6 +386,7 @@ public String readMicr() {
         }
     }
 
+    // TODO should this be beingInsertion
```

Comment:
I don't think this should be _being_ anything
```suggestion
    // TODO should this be beginInsertion
```